src/database/datatools/datadescription.py

```
@@ -0,0 +1,94 @@
+
+def _gen_description_all_materials(data:list):
+    des = f"这是energyplus中的all_materials下的材料数据，在数据库中的id为{data[0]}，名称为{data[1]}，其材料类型为{data[2]}，如果是standard_material则在standard_materials表中的id为{data[3]}，如果是no_mass_material则在no_mass_materials表中的id为{data[4]}。"
+    return des
+
+
+
+def update_description_material(db_path, data:list):
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+    description = _gen_description_material(data)
+    sql = f"UPDATE standard_materials SET description = ? WHERE id = ?"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)